src/scss/_base.scss

```diff
@@ -36,21 +36,17 @@
 		text-align: left;
 		vertical-align: top;
 	}
-	
+
 	.primary-data {
```

Comment:
I know it's not part of the PR but I think updating these to `o-table__cell--primary` style classnames might be better? @j-mes was there a reason for making them nested/non-BEM? If not I'll open up an issue... / @AlbertoElias